tests/test_build_features.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from src.features.build_features import text_prepare
+
+
+class TestBuildFeatures:
+
+    def test_text_prepare(self):
+        examples = ["SQL Server - any equivalent of Excel's CHOOSE function?",
+                    "How to free c++ memory vector<int> * arr?"]
+        answers = ["sql server equivalent excels choose function",
+                   "free c++ memory vectorint arr"]
+        for ex, ans in zip(examples, answers):
+            if text_prepare(ex) != ans:
+                return "Wrong answer for the case: '%s'" % ex
+        return 'Basic tests are passed.'
```

Comment:
I don't think this test will ever fail since there are no asserts. You could replace the return statements and the if statement by for example `assert text_prepare(ex) != ans:`

Reply:
Aah, i'll change that real quick

Reply:
Fixed it :)